crates/cli-support/src/js/mod.rs

```diff
@@ -833,19 +833,19 @@ impl<'a> Context<'a> {
                     }}
                 }}
 
-                function getImports() {{
+                function wasm_bindgen_getImports() {{
```

Comment:
Could this be called `wasm_bindgen_get_imports` (or, in line with the above comment, `__wbg_get_imports`) instead? Mixed casing conventions like this look weird.

Same for `wasm_bindgen_initMemory`.

Reply:
agreed, done!